app/css/lists.css

```diff
@@ -38,12 +38,19 @@ a {
 .li__comments {
   color: #858585;
   display: inline-block;
-  font-size: 13px;
+  font-size: 1.3rem;
```

Comment:
Let's keep things in px. We can discuss but I've never understood why Gaia is designed with rem.